### PR TITLE
Unit lookup

### PR DESCRIPTION
**Pull request by shimthal61**
Added unit look functionality.

dice.html is mostly untouched, the majority of the code is in unit_lookup.js.

Created `public` folder to facilitate node.js local server hosting. 

Moved wh40k-10e and dice files into public folder.

### CODE REVIEW

File: 40kdice/public/dice.html [hunk 1]
```diff
@@ -87,12 +88,30 @@ <h2>Attack Stats</h2>
             <option value="">No wound rolls</option>
             <option value="1">Wound rolls of 1</option>
             <option value="fail">Failed wound rolls</option>
-            <option value="noncrit">Non-critical wound rolls</option>
+            <option value="noncrit">Non-critical wound r∆olls</option>
```
Comment (lukaross368) [inline 40kdice/public/dice.html]: typo?

Comment (shimthal61) [reply 40kdice/public/dice.html]: yeah typo

File: .DS_Store
Comment (lukaross368) [inline .DS_Store:1]: remove .DS_store file, we should create a .gitignore and make sure .DS_Store is included

**Review by lukaross368 — APPROVED** [review]

**Comment by lukaross368** [comment]
Looks good to merge after conflicts and .DS_Store resolved!

**Pull request merged**